gnat2goto/driver/tree_walk.adb

```diff
@@ -1036,8 +1037,7 @@ package body Tree_Walk is
    begin
       Set_Source_Location (Ret, Sloc (N));
       Set_Type (Ret, Real_Constant_Type);
-      --  ??? FIXME
-      Set_Value (Ret, "0");
+      Set_Value (Ret, Convert_Ureal_To_Binary_IEEE (Realval (N)));
```

Comment:
Does this handled the difference between floating point and fixed point real types?